### PR TITLE
use fqdn for all image names when dockerhub is the registry

### PR DESCRIPTION
**Pull request by mukundansundar**
Signed-off-by: Mukundan Sundararajan <65565396+mukundansundar@users.noreply.github.com>

# Description

Use FQDN when Dockerhub is set as the default regirstry.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1090 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: pkg/standalone/standalone.go [hunk 1]
```diff
@@ -52,18 +52,23 @@ const (
...
-	githubContainerRegistryName = "ghcr"
-	ghcrURI                     = "ghcr.io/dapr"
-	daprGhcrImageName           = "dapr"
-	redisGhcrImageName          = "3rdparty/redis:6"
-	zipkinGhcrImageName         = "3rdparty/zipkin"
+	dockerURI = "docker.io"
+	ghcrURI   = "ghcr.io"
+
+	// used when DAPR_DEFAULT_IMAGE_REGISTRY is set as GHCR or image-registry flag is set.
+	daprGhcrImageName   = "dapr/dapr"
```
Comment (artursouza) [inline pkg/standalone/standalone.go:69]: Why not the same here?

Comment (mukundansundar) [reply pkg/standalone/standalone.go:69]: this one is used for build the private registry image fqdn or the `ghcr.io` registry fqdn ...
finally the names will be `ghcr.io/dapr/dapr` or `ghcr.io/dapr/3rdparty/zipkin` etc. or `<private registry>/dapr/dapr` ...

**Review by artursouza — APPROVED** [review]

File: pkg/standalone/standalone.go (hunk 1 above)
Comment (pravinpushkar) [inline pkg/standalone/standalone.go:69]: Should we make this also FQDN and have separate for image-registry ?

Comment (mukundansundar) [reply pkg/standalone/standalone.go:69]: the image name is concatenated and made FQDN when ghcr is selected or a private registry is given ...

**Review by pravinpushkar — APPROVED** [review]
lgtm

**Comment by matzew** [comment]
thanks @pravinpushkar  (see #1180)

**Pull request merged**